patches/net/minecraft/client/gui/screens/MenuScreens.java.patch

```diff
@@ -19,19 +19,19 @@
      }
  
      @Nullable
-@@ -54,6 +_,11 @@
+@@ -54,13 +_,23 @@
          return (MenuScreens.ScreenConstructor<T, ?>)SCREENS.get(p_96200_);
      }
  
+-    private static <M extends AbstractContainerMenu, U extends Screen & MenuAccess<M>> void register(
 +    /**
-+     @deprecated Use {@link net.neoforged.neoforge.client.event.RegisterMenuScreensEvent} to register menu screens
-+     + TODO 1.20.5: remove AT related to this
++     * @deprecated use {@link net.neoforged.neoforge.client.event.RegisterMenuScreensEvent} instead
 +     */
 +    @Deprecated
-     public static <M extends AbstractContainerMenu, U extends Screen & MenuAccess<M>> void register(
++    public static <M extends AbstractContainerMenu, U extends Screen & MenuAccess<M>> void register(
```

Comment:
Patch is bit screwed up, it's currently patching back in the AT - you'll need to change the access modifier in the `:neoforge` project and regen patches

Reply:
do we even need `@Deprecated` anymore? Because modders won't be able to use this anymore.